src/AppRoutes.jsx

```diff
@@ -29,10 +29,12 @@ const AppRoutes = () => {
         element={session.info.isLoggedIn ? <Navigate to={path} replace /> : <Home />}
       />
       <Route element={<ProtectedRoute isLoggedIn={session.info.isLoggedIn} />}>
-        <Route path="/clients" element={<Clients />} />
-        <Route path="/documents" element={<Documents />} />
+        <Route path="/clients">
+          <Route index element={<Clients />} />
+          <Route path=":id" element={<Profile />} />
```

Comment:
This won't work. The ID you're using is the id of the row in the clients list table. If we sort the row differently, or we add a new client, or we remove a client, those IDs will change. And a route that once referred to one client will now refer to a different client. You'll need to use the id _of the client themselves_. That would be their web ID. You can pass the web ID as a query param to the route. See my previous comment about what these can look like and how to use query params.

If we use web IDs, we won't need separate routes for the logged in user profile and other people's profiles. They can all fall under a `/profiles?webId="webId"` route.

Reply:
I'll take another crack at it.

Reply:
Not having much luck with useSearchParam unfortunately. Managed to get the URL changed with the query (i.e. the URL becomes http://localhost:5173/clients?webId=https%3A%2F%2Fnewtestuser.opencommons.net%2Fprofile%2Fcard%23me), but couldn't manage to get the profile page to actually load.

If I tried to re-route the page with navigator or Link, the seems to query revert to null. The browser url seems to flicker, but I see the page changing to http://localhost:5173/clients?null before going back to http://localhost:5173/clients.

For now, to avoid routing based on `labelId`, I'll use the client's "webId" instead after processing it with `encodedURIComponent` (was thinking to use username, but think webId would be better).

https://github.com/codeforpdx/PASS/assets/14917816/632b9679-83b8-48d8-ae76-9e07c3e45fc5

This seems to work?